Formula/b/bitcoin.rb

```diff
@@ -46,16 +46,6 @@ class Bitcoin < Formula
     cause "Requires std::filesystem support"
```

Comment:
Note that we now also require GCC 10+, due to C++20 usage, so this should also be updated.